Bottom: d21ed0ee2d0e358a08e1926537cc46ada62529c4
Top:    32a4f1e57c822647aaea85798ab17269348c3ad7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 20:04:19 +0000

Refresh of add-flag-parameters-to-resetfo

---

diff --git a/src/find.c b/src/find.c
index 3248947..d03bc8c 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3100,7 +3100,7 @@ LookupElementConnections (ElementType *Element, FILE * FP)
   /* reset all currently marked connections */
   User = true;
   TheFlag = FOUNDFLAG;
-  ResetConnections (true, TheFlag);
+  ResetConnections (true, FOUNDFLAG);
   InitConnectionLookup ();
   PrintElementConnections (Element, FP, true);
   SetChangedFlag (true);
@@ -3121,7 +3121,7 @@ LookupConnectionsToAllElements (FILE * FP)
   /* reset all currently marked connections */
   User = false;
   TheFlag = FOUNDFLAG;
-  ResetConnections (false, TheFlag);
+  ResetConnections (false, FOUNDFLAG);
   InitConnectionLookup ();
 
   ELEMENT_LOOP (PCB->Data);
@@ -3131,12 +3131,12 @@ LookupConnectionsToAllElements (FILE * FP)
       break;
     SEPARATE (FP);
     if (Settings.ResetAfterElement && n != 1)
-      ResetConnections (false, TheFlag);
+      ResetConnections (false, FOUNDFLAG);
   }
   END_LOOP;
   if (Settings.RingBellWhenFinished)
     gui->beep ();
-  ResetConnections (false, TheFlag);
+  ResetConnections (false, FOUNDFLAG);
   FreeConnectionLookupMemory ();
   Redraw ();
 }
